docs/application/web/guides/sensors/ham.md

```diff
@@ -85,7 +85,22 @@ Enabling the monitor and retrieving data is a basic Human Activity Monitor (HAM)
    tizen.humanactivitymonitor.getHumanActivityData('HRM', onsuccessCB, onerrorCB);
    ```
 
-4. To disable HAM when it is no longer required, use the `stop()` method of the `HumanActivityMonitorManager` interface:
+4. If requested `HumanActivityType` type is `PEDOMETER`, then `getHumanActivityData()` provides `HumanActivityPedometerData` object data gathered since the call of the `start()` method, but also data, which is gathered since the device boot.
```

Comment:
Request you to do the slight modification for more clarity: 
- Also as the sentence is followed by code snippet, we have to replace "." with : 
 
```suggestion
4. If the requested `HumanActivityType` type is `PEDOMETER` then getHumanActivityData() provides the `HumanActivityPedometerData` object data. The object data is not only gathered since the call of the `start()` method but also since the device boot:
```

Reply:
Done

Reply:
@pkosko 
Thanks ! but kindly incorporate the following changes too. 

4. If the requested `HumanActivityType` type is `PEDOMETER` then `getHumanActivityData()` provides the `HumanActivityPedometerData` object data. The object data is not only gathered since the call of the `start()` method but also since the device boot:

Reply:
I'm sorry about that. I missed your suggestion. I think that the meaning of the sentence change after your modification. Please share your opinion about below:
```suggestion
4. If the requested `HumanActivityType` type is `PEDOMETER` then `getHumanActivityData()` provides the `HumanActivityPedometerData` object. Object includes data not only gathered since the call of the `start()` method, but also data, which is gathered since the device boot:
```

Reply:
Thanks @pkosko for verifying the meaning and providing a new rephrase. 
- Please share your valuable opinion about the following modification. 
- Also request you to verify the technical correctness/meaning.

```suggestion
4. If the requested `HumanActivityType` type is `PEDOMETER` then `getHumanActivityData()` provides the `HumanActivityPedometerData` object. This object not only includes data gathered since the call of the `start()` method, but also the data gathered since the device boot:
```

Reply:
@nawab-samsung you are right, the original sentence was not what it should be and because of that after your rephrasing it didn't meet the correct meaning also.
I applied your final suggestion, I think it well describes the result of this step of scenario.

Thank you for all suggestions!

Reply:
@pkosko Thanks a lot for your words, really I appreciate it.